enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/busted/Busted.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.store.format.busted;
+
+import org.neo4j.kernel.impl.store.format.RecordFormat;
+import org.neo4j.kernel.impl.store.format.RecordFormats;
+import org.neo4j.kernel.impl.store.format.lowlimit.LabelTokenRecordFormat;
+import org.neo4j.kernel.impl.store.format.lowlimit.PropertyKeyTokenRecordFormat;
+import org.neo4j.kernel.impl.store.format.lowlimit.RelationshipTypeTokenRecordFormat;
+import org.neo4j.kernel.impl.store.record.DynamicRecord;
+import org.neo4j.kernel.impl.store.record.LabelTokenRecord;
+import org.neo4j.kernel.impl.store.record.NodeRecord;
+import org.neo4j.kernel.impl.store.record.PropertyKeyTokenRecord;
+import org.neo4j.kernel.impl.store.record.PropertyRecord;
+import org.neo4j.kernel.impl.store.record.RelationshipGroupRecord;
+import org.neo4j.kernel.impl.store.record.RelationshipRecord;
+import org.neo4j.kernel.impl.store.record.RelationshipTypeTokenRecord;
+
+/**
+ * Record format with very high limits, 58-bit per ID, while at the same time keeping store size small.
+ *
+ * @see BaseBustedRecordFormat
+ */
+public class Busted implements RecordFormats
```

Comment:
Can we consider to rename this format to something that will bring more descriptive name and highlight is dynamic size nature? because we will forget what `busted` has to do with in in couple of months